core/services/chainlink/config_test.go

```diff
@@ -1331,7 +1331,9 @@ func TestConfig_Validate(t *testing.T) {
 		- 1.ChainID: invalid value (1): duplicate - must be unique
 		- 0.Nodes.1.Name: invalid value (foo): duplicate - must be unique
 		- 3.Nodes.4.WSURL: invalid value (ws://dupe.com): duplicate - must be unique
-		- 0: 3 errors:
+		- 0: 5 errors:
+			- logbroadcaster cannot be enabled unless all nodes have WSURL provided
+			- Nodes: missing: must have at least one ws uri when LogBroadcaster is enabled
```

Comment:
These seem contradictory. Must all nodes have WSURL, or just one?

Reply:
Thanks for catching it. According to the requirement, I was intended to enforce `Must all nodes have WSURL when LogBroadcaster is enabled`, so the second error message is incorrect. But I will verify the requirement is correct.

Reply:
Updated the error message. We going to require all primary nodes to provide valid WS URL when LogBroadcaster is enabled.